marketplace_confg/schema.yaml

```diff
@@ -57,131 +57,135 @@ properties:
     description: (CStore) IP of DIMSE peer.
   export.peer_dimse_port:
     type: integer
-    title: "exporter - Peer DIMSE port"
+    title: "Export Adapter: Peer DIMSE port"
     description: (CStore) Port of DIMSE peer.
   export.project_id:
     type: string
-    title: "exporter - Pubsub project id"
+    title: "Export Adapter: Pubsub project id"
     description:  >-
       Pubsub project id, must be the same as the project id in which the adapter is running.
   export.subscription_id:
     type: string
-    title: "exporter - Pubsub subscription id"
+    title: "Export Adapter: Pubsub subscription id"
     description: The pubsub subscription id
   export.dicomweb_addr:
     type: string
-    title: "exporter - Dicom web address"
+    title: "Export Adapter: DICOMweb address"
     description: >-
       Base api endpoint. e.g. 
       https://healthcare.googleapis.com/v1
   export.oauth_scopes:
     type: string
-    title: "exporter - Oauth scopes"
+    title: "Export Adapter: Oauth scopes"
     description: Comma seperated OAuth scopes used by adapter.
     default: "https://www.googleapis.com/auth/pubsub"
   export.peer_dicomweb_address:
     type: string
-    title: "exporter - Peer dicomweb address"
+    title: "Export Adapter: Peer DICOMweb address"
     description:  >-
-      (StowRs) Address of peer DicomWeb API serving STOW-RS. Must be a full path up 
+      (StowRs) Address of peer DICOMweb API serving STOW-RS. Must be a full path up 
       to /dicomWeb if the Cloud Healthcare API is used. e.g. 
       https://healthcare.googleapis.com/v1/projects/myproject/locations/
       us-central1/datasets/mydataset/dicomStores/mydicomstore/dicomWeb
   export.use_gcp_application_default_credentials:
     type: boolean
-    title: "exporter - Enable gcp application default credentials"
+    title: "Export Adapter: Enable gcp application default credentials"
     description:  >-
       (StowRs) If true, uses GCP Application Credentials to when sending HTTP requests 
-      to peer. This is useful if the peer DICOMWeb endpoint is running in GCP.
+      to peer. This is useful if the peer DICOMweb endpoint is running in GCP.
     enum:
     - true
     - false
   export.monitoring_project_id:
     type: string
-    title: "exporter - Monitoring project id (Optional)"
+    title: "Export Adapter: Monitoring project id (Optional)"
     description:  >-
       (Optional) Stackdriver monitoring project id, must be the same as the project 
       id in which the adapter is running.
   export.custom_flags:
       type: string
-      title: "Extra export adapter flag (Optional)"
+      title: "Extra export adapter flags (Optional)"
       description: >-
-        (Optional) Addtional flag for export adapter.
+        (Optional) Addtional flags for the export adapter.
         e.g. "--flag_a=valueA"
 
 required:
 - name
-- namespace
\ No newline at end of file
+- namespace
+- import.dimse_aet
+- export.peer_dimse_aet
```

Comment:
We don't know if the user will want to deploy an import or export adapter, so I don't think we should have these as required. Especially not the export peer dimse aet since very few customers use the export adapter.

Reply:
I made these required because I don't believe the deployer allows us to deploy each individually, I think both import and export images are  deployed at the same time.  If these are not set, when deploying an error is thrown. Let me know if you still want me to mark these as required?

Reply:
Removed this requirement.